src/main/js/main_html5.js

```diff
@@ -855,6 +855,31 @@ require("../../../html5-common/js/utils/environment.js");
       }
     };
 
+    /**
+     * For multi audio we can get a list of available audio tracks
+     * @public
+     * method OoyalaVideoWrapper#getAvailableAudio
+     * @returns {(boolean|Array)} false - if an array with tracks is not available or Array - an array of all available audio tracks.
```

Comment:
Can we make it so that this function only returns a single type? If there is no available audio, I believe an empty array should suffice. With a consistent return type, we would not have to worry about js errors when we call <returned array>.length, for example.

Reply:
\+